IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ElectricFlowStorageDeviceTypeBattery/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties of batteries used in railway. The property set can be used by the predefined type BATTERY of IfcElectricFlowStorageDevice.
```

Comment:
Only valid for railway?

Reply:
0871f7b28802c8b6bf8a2b767b0356cb11e25edd

Reply:
Not only. 0871f7b28802c8b6bf8a2b767b0356cb11e25edd